test/commons/text/form-control-value.js

```diff
@@ -408,30 +408,30 @@ describe('text.formControlValue', function() {
       '  <div role="option" aria-selected="true">bar</div>' +
       '</div>';
 
-    it('returns the text of role=combobox elements', function() {
+    it('returns the text of role=combobox elements', function () {
       var target = queryFixture(
         '<div id="target" role="combobox">' + comboboxContent + '</div>'
       );
       assert.equal(ariaComboboxValue(target), 'bar');
     });
 
-    it('returns `` for elements without role=combobox', function() {
-      queryFixture('<div role="combobox">' + comboboxContent + '</div>');
+    it('returns `` for elements without role=combobox', function () {
+      fixtureSetup('<div role="combobox">' + comboboxContent + '</div>');
```

Comment:
This would be better written by using `fixtureSetup` and passing a custom query. I can't suggest the next 4 lines, so this should replace the `querySelectorAll` for the `[role=listbox]` as well

```js
var target = queryFixture('<div role="combobox">' + comboboxContent + '</div>', '[role=listbox]');
```